docs/changes/coordinates/17824.feature.rst

```diff
@@ -0,0 +1,3 @@
+``search_around_sky`` now accepts a list of separations with the same number of
```

Comment:
Using the word "list" here is needless source of potential confusion.

Reply:
This now says: ``search_around_sky`` and ``search_around_3D`` now accept separations/distlimits
broadcastable to the same shape as ``coords1``.